src/finch/tensor.py

```diff
@@ -245,12 +261,12 @@ def __getitem__(self, key):
         if not isinstance(key, tuple):
             key = (key,)
 
-        if None in key:
+        if not self.is_computed():
```

Comment:
I think we still want to support lazy indexing in eager mode, why we need this change?

Reply:
I've added support for `None` in eager indexing in this commit (after this comment): https://github.com/finch-tensor/finch-tensor-python/pull/97/commits/e4bd64c35d1c6d57b169270e633d65ceb64907aa